### ISSUE TITLE
Plugin doesn't see related docsets for mutiple docsests

### ISSUE THREAD

**Issue by petRUShka** [sunaku/vim-dasht#11]
Config:
```
:let g:dasht_filetype_docsets
g:dasht_filetype_docsets {'python': ['(num|sci)py', 'pandas']}
```
Current file:
```
:se ft
filetype=python.pyopencl
```

Docsets:
```
# dasht-docsets
LaTeX
NumPy
Python_3
```

vimrc
```
" search related docsets
nnoremap <silent> <Leader>K :call Dasht([expand('<cword>'), expand('<cWORD>')])<Return>
```

In VIM (after `<Leader>K`):

```
dasht: 'astype' not found in python pyopencl docsets
dasht: '.astype' not found in python pyopencl docsets 
```

Latest plugin from git.

**Comment by ghost** [comment]
I am also experiencing this issue with "javascript.jsx" filetypes when developing in React.

**Comment by ghost** [comment]
Please see my PR here: https://github.com/sunaku/vim-dasht/pull/12

**Comment by sunaku** [comment]
@kitanata Thanks for the PR, which showed me where the problem was. 👍 

Please try the `issue-11` branch and confirm whether it solves the issue.

**Comment by ghost** [comment]
@sunaku I just pulled the `issue-11` branch and tested things out. It works well. Your solution is much better than mine. Also, thanks for awesome plugin and dasht itself. I'm really loving it. :)

**Comment by sunaku** [comment]
Great! 👍 Fix pushed to master in commit eb2c3d0288713c959a6c0d60101e42e60c2915ea.
